snoop/defaultsettings.py

```diff
@@ -88,3 +89,5 @@
 celery.app.conf.task_routes = {
     'snoop.data.tasks.check_if_idle': {'queue': 'watchdog'}
 }
+
+LANGUAGE_DETECTOR = language_detection.detectors[LANGUAGE_DETECTOR_NAME]
```

Comment:
I think the django settings should only contain simple values, not references to python objects. I'd move this code to `snoop/data/indexing.py`.

Reply:
fixed